no_drama/pip_automation.py

```diff
@@ -14,9 +15,13 @@ def save_wheels(destination, packages=[], requirements_paths=[]):
     #
     # python -m pip wheel --wheel-dir=dest p1 p2 p3 -r req1.txt -r req2.txt
     # https://pip.pypa.io/en/stable/user_guide/#using-pip-from-your-program
+    environment = os.environ.copy()
+    if pythonpaths:
+        environment['PYTHONPATH'] = ':'.join(pythonpaths)
```

Comment:
This trick doesn't seem to actually work-- the idea was to invoke pip in an environment that includes the bootstrap wheels. If I hack in a call to PDB in the destination python, `import wheel` seems to work, but pip still fails with bdist_wheel not found errors.